src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -164,9 +169,21 @@ static ucs_status_t uct_rdmacm_cm_id_to_dev_addr(struct rdma_cm_id *cm_id,
         return UCS_ERR_NO_MEMORY;
     }
 
+    if (is_eth) {
+        /* pack the local remote RoCE version assuming that rdmacm guarantees that
+         * the remote version is the same */
+        status = uct_ib_device_query_gid_info(cm_id->verbs, dev_name,
```

Comment:
does it mean we may parse sysfs in every connection establishment?

Reply:
yes, for RoCE